app/src/main/java/org/oppia/app/administratorcontrols/appversion/AppVersionViewModel.kt

```diff
@@ -16,7 +17,10 @@ class AppVersionViewModel @Inject constructor(
   private val oppiaDateTimeFormatter: OppiaDateTimeFormatter
 ) : ObservableViewModel() {
 
-  val versionName = ObservableField<String>(BuildConfig.VERSION_NAME)
+  private lateinit var context: Context
```

Comment:
Is this being initialized anywhere? As it stands, the code below will always fail because context is uninitialized.

Reply:
Oh geez. Okay, fixing that now! Sorry!